documentation/adls/sol_refactoring.md

```diff
@@ -0,0 +1,36 @@
+### Title
+Sol refactoring, with lua object caching.
+
+### What was/is the problem?
+Multiple problems, explained in detail [here](https://github.com/topaz-next/topaz/wiki/Sol-Refactor).
```

Comment:
you kept referencing old projects wiki D:

https://github.com/LandSandBoat/server/wiki/Sol-Refactor